src/App.js

```diff
@@ -36,9 +46,9 @@ function App() {
     
     <div>
       <Header showAllPosters={showAllPosters}/>
+      {error && <div><p>{error}</p></div>}
```

Comment:
Do yall need this extra div here? Sometimes when HTML is super nested I can get lost when applying styling or inspecting in my dev tools